tests/unit/parsing/test_git_rs.py

```
@@ -0,0 +1,150 @@
+    (repo_dir / "README.md").write_text("# Test Project\n")
+    (repo_dir / "Makefile").write_text("all:\n\techo done\n")
+
+    src = repo_dir / "src"
+    src.mkdir()
+    (src / "main.rs").write_text("fn main() {}\n")
+
+    lib = src / "lib"
+    lib.mkdir()
+    (lib / "utils.py").write_text("def helper(): pass\n")
+
+    _ = subprocess.run(["git", "init"], cwd=repo_dir, check=True, capture_output=True)
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)